webapp/src/Service/ImportExportService.php

```diff
@@ -1094,7 +1094,7 @@ protected function importAccountsTsv(array $content, string &$message = null): i
                     // Note that https://ccs-specs.icpc.io/ccs_system_requirements#accountstsv
                     // assumes team accounts of the form "team-nnn" where
                     // nnn is a zero-padded team number.
-                    $teamId = preg_replace('/^[^0-9]*0*([0-9]+)$/', '\1', $line[2]);
+                    $teamId = preg_replace('/^[^0-9]*0*([0-9]+)[^0-9]*$/', '\1', $line[2]);
```

Comment:
You probably want to update the comment above

Reply:
Done